File: Python/Estructuras de control/Estructuras condicionales/IntercambiarValores.py

# Realizar un algoritmo que permita intercambiar entre si los valores de dos
# variables A y B.

# Entradas
a = float(input("Ingrese el valor de la variable A: "))
b = float(input("Ingrese el valor de la variable B: "))

# Procesos
temp = a
a = b
b = temp

# Salidas
print(f"El valor de A es: {a}")
print(f"El valor de B es: {b}")

File: C++/Estructuras de control/Estructuras condicionales/IntercambiarValores.cpp

// Realizar un algoritmo que permita intercambiar entre si los valores de dos
// variables A y B.

#include <iostream>
using namespace std;

int main() {
    // Entradas
    float a, b;
    cout << "Ingrese el valor de la variable A: ";
    cin >> a;
    cout << "Ingrese el valor de la variable B: ";
    cin >> b;

    // Procesos
    float temp = a;
    a = b;
    b = temp;

    // Salidas
    cout << "El valor de A es: " << a << endl;
    cout << "El valor de B es: " << b << endl;

    return 0;
}